test/fail_compilation/diag14875.d

```diff
@@ -36,27 +36,24 @@ template Baz(T)
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/diag14875.d(47): Deprecation: class `diag14875.Dep` is deprecated
-fail_compilation/diag14875.d(51): Deprecation: variable `diag14875.depVar` is deprecated
+fail_compilation/diag14875.d(44): Deprecation: class `diag14875.Dep` is deprecated
```

Comment:
Actually looking at the issue, I think we should just merge both test cases, since this does not really test the original issue anymore.